packages/react-core/src/components/Title/examples/TitleSizes.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Title, TitleSizes } from '@patternfly/react-core';
+
+export const CustomTitleSizes: React.FunctionComponent = () => (
```

Comment:
Same as the above comment.

```suggestion
export const TitleCustomSizes: React.FunctionComponent = () => (
```